package.json

```diff
@@ -17,8 +17,8 @@
     "selenium-install": "selenium-standalone install",
     "test": "npm run test-unit && npm run test-regression",
     "test-unit": "mocha tools/testSetup.js \"./test/unit/*.js\" --exit",
-    "test-regression": "gemini test ./test/regression/*js",
-    "test-regression-update": "gemini update ./test/regression/*js",
+    "test-regression": "hermione",
```

Comment:
what's the purpose of the `hermione/screens` directory?

Reply:
When you run test-regression command hermione will make a screenshot of test results and it will compare it with screens in hermione/screens if screens are the same the test will pass otherwise it fails